### PR TITLE
Document `@mtkmodel` and `@connector`

### PR DESCRIPTION
**Pull request by ven-k**
Documents `Model`, `@mtkmodel` and `@connector`

### CODE REVIEW

File: src/systems/model_parsing.jl [hunk 1]
```diff
@@ -364,6 +383,7 @@ function parse_extend!(exprs, ext, dict, body, kwargs)
             vars = nothing
             if Meta.isexpr(b, :(=))
                 vars = a
+                @info 379 typeof(vars.head)
```
Comment (YingboMa) [inline src/systems/model_parsing.jl:386]: This needs to be removed.

Comment (ven-k) [reply src/systems/model_parsing.jl]: Resolved [here](https://github.com/SciML/ModelingToolkit.jl/compare/fe7d99ade59808c4c1e42986ec5f24ef4a8c8cea..b9157698991288753ee02055b40d4f4c4c4a9f4c)

**Review by YingboMa — APPROVED** [review]
Do you mind to also update the rest of the tutorials in MTK?

**Comment by ven-k** [comment]
Will take a look.

**Pull request merged**